app/helpers/vacancies_helper.rb

```diff
@@ -16,10 +16,13 @@ def link_to_sort_by(title, column:, order:, sort:)
       order = sort.reverse_order
       active_class = ' active'
     end
-    link_to title,
-            jobs_path(vacancy_params(sort_column: column,
-                                     sort_order: order)),
-            class: "sortby--#{order}#{active_class || ''}"
+    link_to jobs_path(
+      vacancy_params(sort_column: column, sort_order: order)
+    ), class: "sortby--#{order}#{active_class || ''}" do
+      concat(content_tag('span', 'Sort jobs by ', class: 'visually-hidden'))
+      concat((content_tag 'span', title))
```

Comment:
I'm a bit confused with the use of the parenthesis here. Can we make it consistent across all lines? `concat(content_tag( < content> ) )`

Reply:
Weird, surprised it worked with those inconsistencies :/